docs/source/user-guide/nlp/quickstarts/plot_getting_started.py

```diff
@@ -0,0 +1,213 @@
+# -*- coding: utf-8 -*-
+"""
+Quickstart - Getting Started
+****************************
+
+In order to run deepchecks for nlp all you need to have are the following for both your train and test data:
+
+1. Your text data - a list of strings, each string is a single sample (can be a sentence, paragraph, document etc.)
+2. Your labels - either a Text Classification label or a Token Classification label
```

Comment:
Capitalization of these words is weird.
You can keep it if they were the links to the relevant docs - you can link each one to the relevant part in the guide